packages/react/src/lib/createComponent.ts

```diff
@@ -42,7 +42,7 @@ const createComponent = <P extends ObjectOf<any> = any>({
   const StardustComponent: CreateComponentReturnType<P> = (props): React.ReactElement<P> => {
     // Stores debug information for component.
     // Note that this ref should go as the first one, to be discoverable by debug utils.
-    const stardustDebug = React.useRef(null)
+    const ref = React.useRef(null)
```

Comment:
```suggestion
    if (process.env.NODE_ENV !== 'production') {
      const ref = React.useRef(null)
    }  
```

Reply:
The `ref` was already there. I am not sure we want to have different shape of the `createComponent` in prod vs dev.